flink-python/pyflink/datastream/tests/test_stream_execution_environment.py

```diff
@@ -22,9 +22,12 @@
 
 import unittest
 
+
```

Comment:
Please be careful about these unnecessary changes.

Reply:
Thank you, I will pay more attention to it.